parm/FV3LAM_wflow_nonDA.xml

```diff
@@ -481,5 +486,36 @@ MODULES_RUN_TASK_FP script.
 
       </task>
   </metatask>
+<!--
+************************************************************************
+************************************************************************
+-->
+  <metatask name="&RUN_PRDGEN_TN;">
```

Comment:
@BenjaminBlake-NOAA, to exclude this task from the non-DA eng test, the if statement like `{%- if run_task_run_prdgen %}` needs to be added to this task block similarly with `make_grid` or `make_orog`. You can set `RUN_TASK_RUN_PRDGEN="TRUE"` by default in config_default.sh and set it to "FALSE" for the non-DA eng tests.

Reply:
That sounds like a good approach to me, I am testing it now and will commit my changes in a few minutes.